article/app/controllers/LiveBlogController.scala

```diff
@@ -413,7 +413,7 @@ class LiveBlogController(
   def getTopMentions(blogId: String, topics: Option[String]) = {
     val topMentionsResult = for {
       topMentionTopic <- TopMentionsTopic.fromString(topics)
-      topMentions <- topMentionsService.getTopMentionsByTopic(blogId, topMentionTopic)
+      topMentions <- topMentionsService.getBlogSelectedTopic(blogId, topMentionTopic)
```

Comment:
I find this a bit confusing - maybe `getSelectedTopic` or `getSelectedTopicDetails` is closer to the functions outcome?

Reply:
yea good point, change it 👍